packages/reactiveData/reactiveData.0.2.4/opam

```diff
@@ -29,6 +29,7 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/ocsigen/reactiveData.git"
+flags: [ avoid-version ]
```

Comment:
```suggestion
flags: [ avoid-version ]
x-maintained: false
```
If you add this, it will be archived at the next occasion and be out of the way

Reply:
Is that a problem ? 0.2.4 was released but there was already an older version in the opam repo with a version greater than 0.2.4. (0.3.0)

Reply:
Not at all. I just wanted to let you know that if there are released that you don't want to maintain or that you want to archive, you can use `x-maintained` for a specific package or `x-maintenance-intent` for the general maintenance strategy